### PR TITLE
adding all changes

### LINKED ISSUE #29: Adding a rating feature to the book store
Hi everyone,

A lot of the users have said that there's no clear way to distinguish the quality of the books in our portal. I was thinking we should have a rating feature or something similar to that to allow users to quickly pick up most highest rated books from the other users.
 - [ ] Get UX to do a quick mock-up
 - [ ] If we decide to go ahead get the design done
 - [ ] Software Development
 - [ ] Testing and Review

Would the UX team care to come up with a quick wireframe?

### PR DESCRIPTION
**Pull request by affrae**
Hello,

This pull request fixes #29   - the new rating feature we discussed on the issue. The main changes are:
- a new property in the data model: `rating`
- Data may contain a rating
- Service is untouched, but had some refactoring
- New image for the star refactoring

Please proceed with a review once the required status checks are valid.

@affraedemo, can you please review?

### CODE REVIEW

File: src/main/java/com/github/demo/model/Book.java [hunk 1]
```diff
@@ -11,19 +11,23 @@
 
     private String cover;
 
+    private long rating;
```
Comment (affraedemo) [inline src/main/java/com/github/demo/model/Book.java:14]: H @affrae I think that a `long` is a little expensive, how about an `int`

File: src/main/java/com/github/demo/model/Book.java [hunk 2]
```diff
@@ -53,4 +57,19 @@ public String getCover() {
     public void setCover(String cover) {
         this.cover = cover;
     }
-}
\ No newline at end of file
+
+    public long getRating() {
```
Comment (affraedemo) [inline src/main/java/com/github/demo/model/Book.java:61]: again int

File: src/main/java/com/github/demo/model/Book.java [hunk 3]
```diff
@@ -53,4 +57,19 @@ public String getCover() {
...
+        return rating;
+    }
+
+    public void setRating(long theRating) {
```
Comment (affraedemo) [inline src/main/java/com/github/demo/model/Book.java:65]: also int

File: src/main/java/com/github/demo/service/BookService.java [hunk 4]
```diff
@@ -10,16 +10,16 @@
...
-        books.add(new Book("Jeff Sutherland","Scrum: The Art of Doing Twice the Work in Half the Time", "scrum.jpg"));
-        books.add(new Book("Eric Ries","The Lean Startup: How Constant Innovation Creates Radically Successful Businesses", "lean.jpg"));
-        books.add(new Book("Geoffrey A. Moore","Crossing the Chasm", "chasm.jpg"));
-        //books.add(new Book("David Thomas","The Pragmatic Programmer: From Journeyman to Master", "pragmatic.jpg"));
-        //books.add(new Book("Frederick P. Brooks Jr.", "The Mythical Man-Month: Essays on Software Engineering", "month.jpg"));
-        books.add(new Book("Steve Krug","Don't Make Me Think, Revisited: A Common Sense Approach to Web Usability", "think.jpg"));
+        books.add(new Book("Jeff Sutherland","Scrum: The Art of Doing Twice the Work in Half the Time", "scrum.jpg", 3));
+        books.add(new Book("Eric Ries","The Lean Startup: How Constant Innovation Creates Radically Successful Businesses", "lean.jpg", 5));
+        books.add(new Book("Geoffrey A. Moore","Crossing the Chasm", "chasm.jpg", 4));
+        //books.add(new Book("David Thomas","The Pragmatic Programmer: From Journeyman to Master", "pragmatic.jpg", 3));
```
Comment (affraedemo) [inline src/main/java/com/github/demo/service/BookService.java:16]: hey - we are using a VCS so how about deleting the comments?

File: src/main/java/com/github/demo/model/Book.java (hunk 1 above)
Comment (affrae) [reply src/main/java/com/github/demo/model/Book.java:14]: I agree I will fix

**Pull request merged**